spec/features/list_of_products_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'List of products', type: :feature do
+  before(:each) do
+    build_list(:product, 11, &:save)
```

Comment:
what's the difference between build_list(:product, 11, &:save) and create_list(:product, 11)?

Reply:
No difference, it works the same way) Changed